### PR TITLE
Update Email deployment scripts

### PR DESCRIPTION
**Pull request by lauren-mills**
## Purpose
*What is the context of this pull request? Why is it being done?*
- Update Email skill to latest deployment scripts

## Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp?*
- Updated deployment scripts to latest from #1722 and #1694 
- Updated properties in appsettings
- Updated casing of luisgen file and lu files to camel casing

*Include illustrative screenshots for context if applicable.*

## Tests
*Is this covered by existing tests or new ones? If no, why not?*
- Updated existing tests

## Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

### CODE REVIEW

**Comment by DingmaomaoBJTU** [comment]
Thanks!:smile:  #Closed

**Review by DingmaomaoBJTU — APPROVED** [review]
:shipit:

File: skills/src/csharp/emailskill/emailskill/Services/EmailLuis.cs [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
+// regenerated.
+// </auto-generated>
 using Newtonsoft.Json;
 using System.Collections.Generic;
 using Microsoft.Bot.Builder;
 using Microsoft.Bot.Builder.AI.Luis;
 namespace Luis
 {
-    public class emailLuis: IRecognizerConvert
+    public partial class EmailLuis: IRecognizerConvert
```
Comment (lzc850612) [inline skills/src/csharp/emailskill/emailskill/Services/EmailLuis.cs:13]: why make it partial class? you didn't do this in the ToDo skill change

**Review by lzc850612 — CHANGES_REQUESTED** [review]

File: skills/src/csharp/emailskill/emailskill/Services/EmailLuis.cs (hunk 1 above)
Comment (lauren-mills) [reply skills/src/csharp/emailskill/emailskill/Services/EmailLuis.cs:13]: I reran the luisgen tool and that is what it does by default. I need to go back and do it for todo as well. I was just aligning to the tools default's

Comment (lauren-mills) [reply skills/src/csharp/emailskill/emailskill/Services/EmailLuis.cs:13]: Made the change in todo

**Review by lzc850612 — APPROVED** [review]

**Pull request merged**